README.md

```diff
@@ -70,6 +70,21 @@ The model also uses the following variables, which are currently all set to nan
 - h_median: The median of the recent pv data over the last 7 days
 - h_max: The max of the recent pv data over the last 7 days
 
+## Running Multiple Forecasts
+To run multiple forecasts and save the results, 
+1. Navigate to the scripts directory
+```bash
+cd scripts
+```
+2. Add new sites to the "sites" dictionary on line 60. 
+
+3. Run 
+```bash
+python generate_multiple_forecasts.py
```

Comment:
This currently wont run,  it needs to be `python scripts/generate_multiple_forecasts.py` otherwise it can't find the dataset folder its trying to save the csvs into.